src/main/kotlin/com/kotlin/boardproject/global/handler/OAuth2AuthenticationFailureHandler.kt

```diff
@@ -1,14 +1,14 @@
-package com.kotlin.boardproject.common.handler
+package com.kotlin.boardproject.global.handler
 
-import com.kotlin.boardproject.repository.common.OAuth2AuthorizationRequestBasedOnCookieRepository
-import com.kotlin.boardproject.repository.common.REDIRECT_URI_PARAM_COOKIE_NAME
+import com.kotlin.boardproject.global.repository.OAuth2AuthorizationRequestBasedOnCookieRepository
```

Comment:
🚫 **[ktlint]** <import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end